pkg/pxc/app/statefulset/node.go

```diff
@@ -59,6 +59,22 @@ func (c *Node) Name() string {
 	return app.Name
 }
 
+func (c *Node) InitContainers(cr *api.PerconaXtraDBCluster, initImageName string) []corev1.Container {
+	inits := []corev1.Container{}
+	if cr.CompareVersionWith("1.5.0") >= 0 {
+		var initResources corev1.ResourceRequirements
+		if cr.CompareVersionWith("1.6.0") >= 0 {
+			initResources = cr.Spec.PXC.Resources
+		}
+		if cr.Spec.InitContainer.Resources != nil {
+			initResources = *cr.Spec.InitContainer.Resources
+		}
+		initC := EntrypointInitContainer(initImageName, app.DataVolumeName, initResources, cr.Spec.PXC.ContainerSecurityContext, cr.Spec.PXC.ImagePullPolicy)
+		inits = append(inits, initC)
+	}
```

Comment:
we can remove these version checks to simplify the code

Reply:
https://github.com/percona/percona-xtradb-cluster-operator/pull/1743/commits/f8545394952bdd75904a101ebb697d6755cfbd81